qiskit/transpiler/preset_passmanagers/level0.py

```diff
@@ -162,6 +164,16 @@ def _direction_condition(property_set):
         else:
             raise TranspilerError("Invalid scheduling method %s." % scheduling_method)
 
+    # 8. Add dynamical decoupling passes if requested
+    if dynamical_decoupling is not None:
+        if scheduling_method is None:
+            _scheduling = ALAPSchedule(instruction_durations)
```

Comment:
Should a warning be raised (or log message emitted) in this case to inform that a scheduling policy has automatically been selected?

Reply:
Ali and Itoko recently opted not to warn on implicit scheduling: https://github.com/Qiskit/qiskit-terra/pull/5563